packages/react-next/src/components/Persona/PersonaPresence/PersonaPresence.test.tsx

```diff
@@ -93,4 +95,11 @@ describe('PersonaPresence', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  isConformant({
+    componentPath: path.join(__dirname, 'PersonaPresence.tsx'),
+    Component: PersonaPresence,
+    displayName: 'PersonaPresence',
+    disabledTests: ['exported-top-level', 'has-top-level-file'],
```

Comment:
This one actually is exported top-level (despite not having a top-level file), so the test shouldn't be disabled. If the test isn't handling that scenario properly we should fix it.